webrtc.html

```diff
@@ -5390,29 +5406,29 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
       </div>
       <section>
         <h4>Processing Remote MediaStreamTracks</h4>
-        <p>An application can reject incoming <code><a>MediaStreamTrack</a></code>
-        objects by calling <code><a>RTCRtpTransceiver</a>.stop()</code>.</p>
-        <p>To <dfn id="process-remote-track">process the remote track</dfn> for
+        <p>An application can reject incoming media descriptions by calling
```

Comment:
Isn't it tracks you can reject rather than the whole media description (SDP)? Or am I getting this wrong?